cloudwatch/cloudwatch.go

```diff
@@ -545,6 +556,9 @@ func (output *OutputPlugin) setGroupStreamNames(e *Event) {
 }
 
 func (output *OutputPlugin) createStream(e *Event) (*logStream, error) {
```

Comment:
`createStream` will always be called but it will fail fast if autoCreateStream is disabled and `output.client.CreateLogStream` won't be called.

Which bit looks weird? I'd be happy to sync to clarify anything